model/lora-interference-helper.cc

```diff
@@ -280,7 +280,6 @@ LoraInterferenceHelper::IsDestroyedByInterference(Ptr<LoraInterferenceHelper::Ev
     Time now = Simulator::Now();
     Time duration = event->GetDuration();
     Time packetStartTime = now - duration;
-    Time packetEndTime = now;
```

Comment:
Is this just removed? why ?

Reply:
The variable was declared but not used, so clang-tidy removed it.

Reply:
Oh, ok then